vcluster/_fragments/sync/versioned-crd-syncing.mdx

```diff
@@ -0,0 +1,44 @@
+import CodeBlock from '@theme/CodeBlock';
+
+## Versioned example
+
+To specify a particular version of a CustomResourceDefinition (CRD), append the version to the resource type, separated by a `/`, using the format `<resource>/<version>` in your configuration.
+
+The specified version must exist in the host cluster when vCluster starts. This version becomes the storage version within the vCluster. When the storage version in the vCluster differs from the host cluster's storage version, vCluster automatically converts custom resources between the two versions.
+
+For example, to use version `v1` of the `certificaterequests.cert-manager.io` CRD, specify:
+
+<CodeBlock language="yaml" title={props.yamlTitle}>
+  {`sync:
+    toHost:
```

Comment:
This part in a partial is odd, since it would be `toHost` on a `fromHost` page

Reply:
Fixed